frontend/src/metabase/notifications/NewPulseSidebar.tsx

```diff
@@ -27,100 +23,38 @@ export function NewPulseSidebar({
   onNewSlackPulse,
 }: NewPulseSidebarProps) {
   const applicationName = useSelector(getApplicationName);
+
   return (
     <Sidebar onCancel={onCancel}>
-      <div className={cx(CS.mt2, CS.pt2, CS.px4)}>
-        <h4>{t`Create a dashboard subscription`}</h4>
-      </div>
-      <div className={cx(CS.my1, CS.mx4)}>
-        <ChannelCard
-          flat
-          className={cx(CS.mt1, CS.mb3, {
-            [cx(CS.hoverParent, CS.hoverInherit)]: emailConfigured,
-          })}
-          isConfigured={emailConfigured}
-          onClick={onNewEmailPulse}
-        >
-          <div className={cx(CS.px3, CS.pt3, CS.pb2)}>
-            <div className={cx(CS.flex, CS.alignCenter)}>
-              <Icon
-                name="mail"
-                className={cx(
-                  CS.mr1,
-                  {
-                    [cx(CS.textBrand, CS.hoverChild, CS.hoverInherit)]:
-                      emailConfigured,
-                  },
-                  { [CS.textLight]: !emailConfigured },
-                )}
-              />
-              <h3
-                className={cx({ [CS.textLight]: !emailConfigured })}
-              >{t`Email it`}</h3>
-            </div>
-            <Text
-              className={cx(CS.textMedium, {
-                [cx(CS.hoverChild, CS.hoverInherit)]: emailConfigured,
-              })}
-            >
-              {!emailConfigured &&
-                jt`You'll need to ${(
-                  <Link
-                    key="link"
-                    to="/admin/settings/email"
-                    className={CS.link}
-                  >
-                    {t`set up email`}
-                  </Link>
-                )} first.`}
-              {emailConfigured &&
-                t`You can send this dashboard regularly to users or email addresses.`}
-            </Text>
-          </div>
-        </ChannelCard>
-        <ChannelCard
-          flat
-          className={cx({
-            [cx(CS.hoverParent, CS.hoverInherit)]: slackConfigured,
-          })}
-          isConfigured={slackConfigured}
-          onClick={onNewSlackPulse}
```

Comment:
I didn't think that I should continue with this pattern.
In the past this `onClick` would **always** fire (even when the channel is not yet configured!). The trick was that `onNewSlackPulse` had an additional check (in a different file) which would return undefined if the channel was not configured.

I made it more explicit in this PR. `onClick` prop is now optional.
1. If the channel is not yet configured, the card will be wrapped in a Link component (links to a relevant section in the admin panel), and this prop does not exist
2. if the channel is configured, the component is not wrapped, and the onClick handler exists and works as before